lib/jekyll-github-metadata/site_github_munger.rb

```diff
@@ -16,9 +16,8 @@ def munge!
         # This is the good stuff.
         site.config["github"] = github_namespace
 
-        return unless should_add_fallbacks?
-        add_url_and_baseurl_fallbacks!
         add_title_and_description_fallbacks!
```

Comment:
:+1: Both are a new feature in themselves, and out of scope for this PR, but 👍 to tackling those in a subsequent pass.